docs/core/encoding.md

```diff
@@ -40,13 +38,10 @@ In the `codec` package, there exists two core interfaces, `Marshaler` and `Proto
 where the former encapsulates the current Amino interface except it operates on
 types implementing the latter instead of generic `interface{}` types.
```

Comment:
I know you didn't write this sentence, but I actually don't understand it.

Is it related to the fact that we changed the signature of some Marshal functions from `interface{}` to `proto.Message`?

Reply:
It's refering to the fact that in particular, `Marshaler` encapsulates methods similar to the [Amino codec](https://godoc.org/github.com/tendermint/go-amino#Codec) but they accept `ProtoMarshaler` instead of `interface{}`